specification/datasets/cost_and_usage/providercalculatedsplitcostallocation.md

```diff
@@ -0,0 +1,24 @@
+# Provider-Calculated Split Cost Allocation
+
+The provider-calculated split cost allocation for provider-defined services is a capability that can be offered by providers which allocates (or in some cases provides more granular detail about) a charge to a more granular level. This is accomplished by taking a charge record present in a FOCUS dataset (origin charge) and splitting it into multiple charge records (allocated charges) to reflect the more granular detail, while ensuring the origin charge can be derived from the combination of allocated charges. This feature is used by practitioners to conduct chargebacks and better understand the usage of resources.
+
+If a provider supports provider-calculated split cost allocation for provider-defined services, a FOCUS dataset MUST adhere to the following requirements:
+
+- The following columns MUST be included in a FOCUS dataset when the provider supports provider-calculated split cost allocation:
+  - [AllocatedResourceDetails](#allocatedresourcedetails)
+  - [AllocatedResourceId](#allocatedresourceid)
+- The combination of allocated charge records MUST match the origin charge record.
+  - The sum of all allocated charges MUST have the same value as to origin charge for all metric columns.
```

Comment:
I suggest somehting like: 
“Allocated Charge Records in the Dataset MUST sum to the corresponding Origin Charge Record for all metric Columns.”